src/main/java/net/ccbluex/liquidbounce/injection/mixins/minecraft/render/MixinEntityRenderer.java

```diff
@@ -98,11 +99,17 @@ private void renderLabel(Entity entity, OrderedText text, MatrixStack matrices,
     }
 
     @Inject(method = "renderLabelIfPresent", at = @At("HEAD"), cancellable = true)
-    private void disableDuplicateNametagsAndInjectMobOwners(T entity, Text text, MatrixStack matrices, VertexConsumerProvider vertexConsumers, int light, float tickDelta, CallbackInfo ci) {
+    private void disableDuplicateNametagsAndInjectMobOwners(S state, Text text, MatrixStack matrices, VertexConsumerProvider vertexConsumers, int light, CallbackInfo ci) {
         // Don't render nametags
+        var entity = ((EntityRenderStateAddition) state).liquid_bounce$getEntity();
         if (ModuleNametags.INSTANCE.getRunning() && ModuleNametags.shouldRenderNametag(entity)) {
             ci.cancel();
         }
     }
 
+    @Inject(method = "updateRenderState", at = @At("HEAD"))
+    private void hookInjectEntityIntoState(T entity, S state, float tickDelta, CallbackInfo ci) {
+        ((EntityRenderStateAddition) state).liquid_bounce$setEntity(entity);
+    }
+
```

Comment:
POV: liquidskid if this was his code being ported to legacy
> oh no my code being ported to a different branch of the same project stop it now 2024 now lets paste everything from legacy and act like its fine and we're not contradicting our own thing

Reply:
???